test/containerized_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+import json
+import http.client as http_client
+
+import pytest
+
+from ovirt_imageio._internal import config
+from ovirt_imageio._internal import server
+from ovirt_imageio._internal.units import KiB
+
+from . import testutil
+from . import http
+
+
+TICKET_IMG_UUID = "file"
```

Comment:
TICKET_ID = "ticket-id"?

But do we really need to use the same id? why not use random uuid for every ticket?

Reply:
Since the fixture was in the module scope, I kept it unique. If I change it to a smaller scope, I will create random uuid per ticket.